files/run_worker.sh

```diff
@@ -13,8 +13,10 @@ source /usr/bin/setup_env_in_openshift.sh
 mkdir -p "${PACKIT_HOME}/.ssh"
 chmod 0700 "${PACKIT_HOME}/.ssh"
 pushd "${PACKIT_HOME}/.ssh" || exit
-install -m 0400 /packit-ssh/id_rsa .
-install -m 0400 /packit-ssh/id_rsa.pub .
+# with rootless podman, if id_rsa* files have 0400 mode,
+# in the host, then we can not access them -> fail here
+install -m 0400 /packit-ssh/id_rsa . || exit
```

Comment:
@jpopelka Do you think this `exit`s are ok even in opeshift?

I realized that using rootless podman when my id_rsa* files inside the host have a 0400 mask docker-compose build is not able to load them into the container and it tooks me a while to realize this file were missing.
I think that if we can fail here, debugging is easier.

Reply:
Yes, I think it's OK.

I actually already have a work-in-progress commit that adds `set -e` to the top of the script so that it exits when **any** command fails. I just need to test it.

Reply:
#1912

Reply:
> #1912

I have tested it, and for me it works, it fails if id_rsa* files were not copied. So I have removed my commit. Thanks!